### PR TITLE
feat(UI): added freeCodeCamp podcast link to nav menu

### PR DESCRIPTION
**Pull request by bjklee**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Partially addresses #52417.

Added both the feature and updated nav menu cypress test to include podcast link. Passed cypress test.

### CODE REVIEW

File: cypress/e2e/default/learn/header/universal-navigation.ts
[the remark this answers is not in the document]
Comment (bjklee) [reply cypress/e2e/default/learn/header/universal-navigation.ts:1]: Hi, I finished writing my playwright test and they work on my local machine. When I try to commit my Ís
✖ eslint --max-warnings=0 --cache --fix /Users/brendanlee/Documents/fa2023/freeCodeCamp/e2e/header.spec.ts:

<--- Last few GCs --->

[8121:0x160008000]    16892 ms: Mark-Compact (reduce) 2046.2 (2085.1) -> 2045.5 (2085.6) MB, 910.96 / 0.00 ms  (average mu = 0.233, current mu = 0.003) allocation failure; scavenge might not succeed


<--- JS stacktrace --->

FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory

Do you know what might be causing this issue?
Screenshot 2023-12-04 at 7 37 20 PM

[the remark this answers is not in the document]
Comment (naomi-lgbt) [reply cypress/e2e/default/learn/header/universal-navigation.ts:1]: Good ol' ESLint running out of memory. Usually see this when there's extraneous files committed, such as during a malformed rebase.

[the remark this answers is not in the document]
Comment (bjklee) [reply cypress/e2e/default/learn/header/universal-navigation.ts:1]: Hm, I am only committing one file. What exactly is a malformed rebase? Also I think you helped me out yesterday on discord I quickly wanted to say thanks :P.

**Review by huyenltnguyen — APPROVED** [review]
LGTM.

I'm not sure about the Chinese and Spanish versions though. I assume a staff member would need to make the change in Crowdin. (And I've update the PR description to prevent the issue thread from being closed)

**Comment by naomi-lgbt** [comment]
Links are one of only two files that ARE manually translated on GitHub.

I'll merge this and make a follow-up PR after this morning's meetings. :3

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**